src/observable/FromEventObservable.ts

```diff
@@ -78,28 +84,31 @@ export class FromEventObservable<T, R> extends Observable<T> {
    */
   static create<T>(target: EventTargetLike,
                    eventName: string,
-                   selector?: (...args: Array<any>) => T): Observable<T> {
-    return new FromEventObservable(target, eventName, selector);
+                   selector?: (...args: Array<any>) => T,
+                   options?: EventListenerOptions): Observable<T> {
+    return new FromEventObservable(target, eventName, selector, options);
```

Comment:
> I am not a big fan of polymorphic methods in javascript

: yes, me too. :) it depends on usecases definitely.

If it's coming, I'd consider overloaded signature similar to what current codebase already have such as `combinlateststatic`, which exports each overloaded type definitions. (https://github.com/ReactiveX/rxjs/blob/5a895e87a670ed2af9435e97c9aca927cb1d492f/src/observable/combineLatest.ts#L9-L74)